### DISCUSSION TITLE
How to move buffer if `edgy.nvim` enabled

### DISCUSSION THREAD

**Discussion by Anifyuli** [LazyVim/LazyVim#2311] [Q&A]
Hello everyone. I have some problem to moving my opened buffer with my mouse after I enable `edgy.nvim` enabled. What should I do to doing it?

**Comment by dpetka2001** [comment]
I don't believe you're able to do that. You can only resize the window using the builtin keys.

**Reply by Anifyuli** [reply]
Oh my. Thanks for your answer
